.vscode/extensions.json

```diff
@@ -1,7 +1,6 @@
 {
     "recommendations": [
         "redhat.vscode-yaml",
-        "dbaeumer.vscode-eslint",
-        "eamodio.tsl-problem-matcher"
```

Comment:
I think it has been updated to `amodio.tsl-problem-matcher`? @itowlson do you want to update it or remove it?

Reply:
Ahh, you're right!
I only looked checked the repo and didn't see any Marketplace links, so I assumed it was dead.

Now that I think about it, last commit being in April 2022 should've given it away. ^-^'

They should really add a link on the README. ^-^'

For now, I can add it back with the new name. If later it's requested to remove, I can just rebase to remove it again.